tools/generator-bot-template-generator/app/templates/generators/app/templates/language-generation/en-us/common.en-us.lg

```diff
@@ -0,0 +1,2 @@
+# WelcomeUser
+- Welcome to the EmptyBot sample
```

Comment:
Would be nice to tweak this to something that reflects the generated bot I guess. Not major.

Reply:
Yeah I wonder if we should even have any dialogs on initial template creation. This essentially outputs an empty-bot-generator and the user presumably will generally replace all the scaffolded empty bot .lg and .lu. I do like the idea of having something there so I will keep it as is. 

As far as templating the name there are issues when the generated generators template file has template variables because the generator generator does not have a definition for those variables in the copyTpl() function but I am going to try something clever. standby.

Reply:
Resolved